packages/styles/scss/components/structured-list/_structured-list.scss

```diff
@@ -28,6 +28,17 @@
     @include focus-outline('outline');
   }
 
+  .#{$prefix}--structured-list-row
+    .#{$prefix}--structured-list-td.#{$prefix}--structured-list-icon-column-first {
+    inline-size: $layout-03;
```

Comment:
I think this <code>$layout-03</code> is a deprecated variable, Can you please check once?